master/internal/api_experiment.go

```diff
@@ -852,36 +904,106 @@ func (a *apiServer) PauseExperiment(
 	}
 }
 
+func (a *apiServer) PauseExperiments(
```

Comment:
can't the singular `PauseExperiment` be converted as well, like `CancelExperiment` or `KillExperiment`? what's different about it?